cmd/ltops/deploy.go

```diff
@@ -30,22 +35,33 @@ var deploy = &cobra.Command{
 		}
 
 		if cluster.Type() == terraform.CLUSTER_TYPE {
-			if len(mattermostFile) == 0 {
+			if len(deployOptions.MattermostBinaryFile) == 0 {
 				return errors.New("required flag \"mattermost\" not set")
 			}
-			if len(loadtestsFile) == 0 {
+			if len(deployOptions.LoadTestBinaryFile) == 0 {
 				return errors.New("required flag \"loadtests\" not set")
 			}
+			if deployOptions.Users > 0 {
+				return errors.New("flag \"users\" not supported for type " + cluster.Type())
+			}
+		} else if cluster.Type() == kubernetes.CLUSTER_TYPE {
+			if len(deployOptions.MattermostBinaryFile) > 0 {
+				return errors.New("flag \"mattermost\" not supported for type " + cluster.Type())
+			}
+			if len(deployOptions.LoadTestBinaryFile) > 0 {
+				return errors.New("flag \"loadtests\" not supported for type " + cluster.Type())
+			}
+			if deployOptions.Users == 0 {
+				return errors.New("required flag \"users\" not set")
+			}
 		}
 
-		err = cluster.DeployMattermost(mattermostFile, licenseFile)
-		if err != nil {
-			return errors.Wrap(err, "Couldn't deploy mattermost")
-		}
+		// TODO: stop hard-coding and add flag when we have multiple profiles
+		deployOptions.Profile = kubernetes.PROFILE_STANDARD
```

Comment:
It would be nice if the profiles worked for both Kubernetes and other cluster types.

Reply:
I'll move the profile constants into the `ltops` package